### DISCUSSION TITLE
Can Bonfhir work with r4 server?

### DISCUSSION THREAD

**Discussion by jagandecapri** [bonfhir/bonfhir#306] [Q&A] [answered]
Hi, can Bonfhir work with r4 server for CRUD, search etc. operations?

**Comment by opiation** [comment, marked as answer]
I believe so! Have a look at our documentation site at http://bonfhir.dev. Here's [a simple example](https://bonfhir.dev/packages/core/fhir-client#crud) of using the `FhirClient` with convenient methods to issue HTTP requests respecting the FHIR R4 specification. That same page also includes examples involving searches and operations. I hope you find those helpful.

Also, I don't know your requirements but consider looking at the `@bonfhir/react` and `@bonfhir/query` packages if you're looking for convenient hooks to accomplish those same tasks but within a React application.

**Reply by jagandecapri** [reply]
Hi @opiation , I'll try it out. Thank you for your response!
